proposals/0315-Add-RPC-Conflict-Management.md

```diff
@@ -3,7 +3,7 @@
 * Proposal: [SDL-0315](0315-Add-RPC-Conflict-Management.md)
 * Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
 * Status: **Returned for Revisions**
-* Impacted Platforms: [ Core ]
+* Impacted Platforms: [ Core / HMI ]
 
 ## Introduction
 This proposal is to add a function to SDL Core that manages concurrent RPC conflicts according to priority. This will allow OEMs to receive the expected request from SDL Core during RPC conflict by modifying the configuration file in which the priority is set.
```

Comment:
Reference to the "configuration file" should be removed, and Introduction should be further updated to reflect revised proposed solution.

Reply:
@theresalech -san,
Thank you for your review again.
I will modify to change from "configuration file" to " configuration table in the policy table".
Does  this way matched your advise?
Also, I will modify to add the "/ Policy Server" onto Impacted Platforms.
Anyway, I will modify this proposal and make a commit it.
Could you please confirm it again?
Thank you.

Reply:
I've provided a suggested change to the `Introduction` section.